Assets/Scripts/UI/InGameUI/PerformanceHUD/FPSRangePerformanceComponent.cs

```diff
@@ -0,0 +1,78 @@
+﻿#region License
+// ====================================================
+// Project Porcupine Copyright(C) 2016 Team Porcupine
+// This program comes with ABSOLUTELY NO WARRANTY; This is free software, 
+// and you are welcome to redistribute it under certain conditions; See 
+// file LICENSE, which is part of this source code package, for details.
+// ====================================================
+#endregion
+using System;
+using UnityEngine;
+using UnityEngine.UI;
+
+/// <summary>
+/// Just measures the max and min range for FPS
+/// </summary>
+public class FPSRangePerformanceComponent : BasePerformanceComponent
+{
+    private const float FPSMeasurePeriod = 0.5f;
+    private const string Display = "{0}: Min\n{1}: Max";
+    private int fpsAccumulator = 0;
+    private float fpsNextPeriod = 0;
+
+    private int currentFps;
+    private int lowestFps = 60;
+    private int highestFps = 60;
+
+    private TextPerformanceComponentUI component;
+
+    public override int priorityID()
```

Comment:
All methods should be PascalCase, so `PriorityID`.

Make sure you run [stylecop](https://github.com/TeamPorcupine/ProjectPorcupine/wiki/StyleCop) on your changes, it will show what code doesn't conform to the project's style guidelines